lib/svg-sprite/transform/svgo.js

```diff
@@ -70,15 +24,22 @@ var SVGO = require('svgo'),
  */
 module.exports = function (shape, config, spriter, cb) {
     config = _.cloneDeep(config);
-    config.plugins = ('plugins' in config) ? defaultPluginConfig.concat(config.plugins) : defaultPluginConfig;
-    config.plugins.push({ removeXMLProcInst: !!spriter.config.svg.xmlDeclaration });
-    config.plugins.push({ removeDoctype: !!spriter.config.svg.doctypeDeclaration });
+    config.plugins = 'plugins' in config ? [...config.plugins] : [];
```

Comment:
Would be nice to confirm if custom SVGO options still work; we don't have a test for this.

Reply:
I'm not too savvy with the spread syntax in JavaScript and might overlook some (wanted) side effects (like implicit type casting?), but wouldn't the following do the same here (now that we drop `defaultPluginConfig`):

```js
config.plugins = config.plugins || []; // assuming config.plugins always is an array
```

or

```
if (!config.hasOwnProperty('plugins')) {
    config.plugins = [];
}
```

Reply:
Hmm, indeed, the spread there might be redundant. It's a pity we don't have tests, so we need to test this out somehow manually for now.